.github/workflows/main.yml

```diff
@@ -55,9 +55,8 @@ jobs:
 
 # ------------------------------------------
 # ------------------------------------------
-# Test client-test-data 
   client-test-data-tests:
-    name: Test client-test-data
+    name: Run client-test-data
```

Comment:
all other names are now lower case

Reply:
Well, if I understood the actual "pattern" (if there was one...) correctly, the name of the action starts with an upper case (it's like the title), the names of the steps with lower case.

Reply:
I didn't spot that pattern, but now that you mention it, it makes sense (to some degree)